index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="author"
+      content="Greta Masliukova, Ricardo Vicente, Ilona Liashenko, Ielyzaveta Shabetnyk, Group2"
+    />
+    <meta name="description" content="The Pink Bear restaurant" />
+    <meta
+      name="keywords"
+      content="Restaurant, Pink Bear, food, food menu, services, website"
+    />
+    <link rel="stylesheet" href="./css/normalize.css" />
+    <link rel="stylesheet" href="./css/styles.css" />
+    <title>The Pink Bear Restaurant</title>
+  </head>
+  <body>
+    <nav id="page-navigation" class="main-nav">
+      <ul>
+        <li>
+          <a href="./index.html"
+            ><img
+              src="./assets/svg/paw-solid.svg"
+              alt="paw-logo"
+              class="paw-logo"
+          /></a>
+        </li>
+        <div class="main-nav__flex">
+          <li><a href="./pages/about.html" class="f-s36">About us</a></li>
+          <li>
+            <a href="./pages/menu.html" class="f-s36">Menu</a>
+          </li>
+          <li>
+            <a href="./pages/application.html" class="f-s36">Careers</a>
+          </li>
+          <li>
+            <button class="primary-btn mr-60">
+              <a href="./pages/book-table.html" class="caps-text f-s32"
+                >Order now</a
+              >
+            </button>
+          </li>
+        </div>
+      </ul>
+    </nav>
+    <header id="header" class="header">
+      <h1>The Pink Bear Restaurant</h1>
+      <img
+        src="./assets/svg/original.svg"
+        alt="pink-bear"
+        class="header__img"
+      />
+    </header>
+    <main class="main-content">
+      <section class="home-page">
+        <img
+          src="./assets/img/we-serve-best-cakes.jpg"
+          alt="man-serving-cake"
+          class="home-page__img img-border"
+        />
+        <section class="home-page__text-button">
+          <h3 class="ff-saira fw700 f-s36 lh-15 bl pl25">
```

Comment:
I would suggest to use h2 tag here instead of h3 to use heading tags in a logical order

Reply:
ok